README.md

```diff
@@ -33,10 +36,14 @@ This repository contains System Design resources which are useful to learn conce
 - [API Gateway](https://www.nginx.com/learn/api-gateway/)
 - [Distributed Locking](https://martin.kleppmann.com/2016/02/08/how-to-do-distributed-locking.html)
 - [Checksum](https://www.lifewire.com/what-does-checksum-mean-2625825)
+- [Micro Frontends](https://newsletter.systemdesign.one/p/micro-frontends)
```

Comment:
"Microfrontends" are almost always one of two things:
- Policy/management issue: Not everyone can access a single large monorepo, so they split it into multiple repos managed by independent teams (possibly including contractors)
- Migration: Project is being migrated gradually, page by page.

I don't think this qualifies under System Design.